README.md

```diff
@@ -146,7 +146,7 @@ You can also do this manually by:
 In order to use the new architecture some extra steps are needed. 
 #### iOS 
 - Install pods with new arch flag inside `example/ios` folder: `RCT_NEW_ARCH_ENABLED=1 pod install`
-- Run `npm run example-ios`
```

Comment:
Let's remove that change.
Having this like so is intentional - it guides how to launch the example from project's root where the README is.
We can add the proposed step as additional, alternative like: "`or run npm run ios`(...)" but still it should be done in separate PR. This PR is about redundant `enabled` property.

Reply:
Done ✔️